src/pages/Browse/index.spec.tsx

```diff
@@ -24,19 +34,64 @@ describe(__filename, () => {
     };
   };
 
-  const render = ({ versionId = '123' } = {}) => {
+  const render = async ({
+    versionId = '123',
+    store = configureStore(),
+  } = {}) => {
     const props = {
       ...createFakeRouteComponentProps({ params: { versionId } }),
     };
 
-    return shallow(<Browse {...props} />);
+    return shallowUntilTarget(<Browse {...props} />, BrowseBase, {
+      shallowOptions: {
+        context: { store },
+      },
+    });
   };
 
-  it('renders a page', () => {
+  it('renders a page', async () => {
     const versionId = '123456';
 
-    const root = render({ versionId });
+    const root = await render({ versionId });
 
     expect(root).toIncludeText(`Version ID: ${versionId}`);
   });
+
+  it('renders a FileTree component', async () => {
+    const version = fakeVersion;
+
+    const store = configureStore();
+    store.dispatch(versionActions.loadVersionInfo({ version }));
+
+    const root = await render({ store, versionId: String(version.id) });
+
+    expect(root.find(FileTree)).toHaveLength(1);
+    expect(root.find(FileTree)).toHaveProp(
+      'version',
+      createInternalVersion(version),
+    );
+  });
+
+  it('calls the API to load the version info on mount', async () => {
```

Comment:
This actually tests two things:

1. That it calls the API to get the version info.
2. That it loads the version into the redux store.

I think this is fine for now, but just wanted to point it out in case you wanted to make it into 2 tests.

It also only tests the happy path, but I guess we're okay with that for now, as this will likely be moved elsewhere?

Reply:
> It also only tests the happy path, but I guess we're okay with that for now, as this will likely be moved elsewhere?

I think we are happy with that now but we should think about handling errors: #182